### PR TITLE
doc: add cozy browser for connector

### PR DESCRIPTION
**Pull request by doubleface**
This is the documentation on how to use Cozy Browser in connectors (jsdom, zombiejs)

### CODE REVIEW

File: src/tutorials/konnector/cozy-browser.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+For some websites, a lot of computation is needed to have a correct authentication and this
+computation is done in javascript. It is possible to do some reverse engineering but there is
+another solution: running a browser simulation in your connector.
+
+## Zombie
+
+We chose a specific js browser implementation which is [zombie](https://github.com/assaf/zombie).
+It is a wraper around jsdom to make it easier for scraping.
```
Comment (drazik) [inline src/tutorials/konnector/cozy-browser.md]:
```suggestion
It is a wrapper around jsdom to make it easier for scraping.
```

File: src/tutorials/konnector/cozy-browser.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+For some websites, a lot of computation is needed to have a correct authentication and this
+computation is done in javascript. It is possible to do some reverse engineering but there is
+another solution: running a browser simulation in your connector.
+
+## Zombie
+
+We chose a specific js browser implementation which is [zombie](https://github.com/assaf/zombie).
```
Comment (drazik) [inline src/tutorials/konnector/cozy-browser.md]:
```suggestion
We choosed a specific JS browser implementation which is [zombie](https://github.com/assaf/zombie).
```

File: src/tutorials/konnector/cozy-browser.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+It is a wraper around jsdom to make it easier for scraping.
+
+## Initialisation
+
+Just add the zombie npm package to your connector using :
```
Comment (drazik) [inline src/tutorials/konnector/cozy-browser.md]:
```suggestion
Add the zombie npm package to your connector using :
```

File: src/tutorials/konnector/cozy-browser.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```
+
+And use our wrapper with proper defaults for connectors :
+
+```js
+const Browser = require('cozy-konnector-libs/dist/CozyBrowser')
+const browser = new Browser()
+```
+
+Then you can use the zombie api to run http requests to the targetted website.
```
Comment (drazik) [inline src/tutorials/konnector/cozy-browser.md]:
```suggestion
Then you can use the zombie API to run HTTP requests to the targeted website.
```

File: src/tutorials/konnector/cozy-browser.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+`browser` will keep its own cookie session.
+
+There is an implementation of the template of connector using the Cozy Browser available :
```
Comment (drazik) [inline src/tutorials/konnector/cozy-browser.md]:
```suggestion
There is an implementation of the connector template using the Cozy Browser available :
```

File: src/tutorials/konnector/cozy-browser.md (hunk 2 above)
Comment (doubleface) [reply src/tutorials/konnector/cozy-browser.md]: I will just pick the JS thing

**Pull request merged**